configs/Asus/P10S-M_WS.conf

```diff
@@ -0,0 +1,60 @@
+# P10S-M WS
+chip   "nct6791-*" 
+    ignore temp1
+    ignore temp2
+    ignore temp3
+    ignore temp4
+    ignore temp6
+    ignore temp8
+    ignore temp9
+    ignore temp10
+    label temp7 "CPU"
+    label temp5 "MB"
+
+    label in0 "VCORE"
+
+    ignore in2 # same as in3
+    ignore in5 # 2040 all the time
+    ignore in11
```

Comment:
Why?

Reply:
Regarding `in11` I can't recall why, it was probably mistake. I will try to figure out what is it and fix it. It's value is between [0.87 and 0.9].

Reply:
Seems I copied it from **H87-Pro** [file](https://github.com/lm-sensors/lm-sensors/blob/70f7e0848410b9ca4dde7abff669bbbecbf137e0/configs/Asus/H87-Pro.conf#L40).

Reply:
It seems in 99% cases it's same as VCCIO (`in9`), but sometimes it could be slightly different (by `0.01`) from it.